packages/atlas/src/views/viewer/MemberView/MemberView.styles.ts

```diff
@@ -31,23 +31,38 @@ export const StyledTabs = styled(Tabs)`
 
 export const TabsContainer = styled.div`
   display: grid;
-  margin-bottom: ${sizes(8)};
+  padding-top: ${sizes(8)};
   gap: ${sizes(2)};
-  grid-template: 'tabs tabs tabs' 1fr 'search search search' auto 'sort sort sort' auto / 1fr 1fr;
+  grid-template: 'tabs tabs tabs' 1fr 'search search search' auto 'sort sort sort' auto / 1fr 1fr 'filter filter filter' auto / 1fr 1fr;
```

Comment:
Something is very wrong on smaller screens: 
![image](https://user-images.githubusercontent.com/51168865/153201011-7b8d1a8f-6670-4fba-a3c3-c5213a09b4fe.png)

Second thing - can we just do this without declaring grid-areas? `  grid-template: 1fr auto auto / 1fr 1fr;`? should work the same in this case, right?

Reply:
Grid areas is just easier due the mobile grid layout difference where tabs take 2 columns, I think they are fine it was just not properly indented/wrong